auslib/test/test_db.py

```diff
@@ -1109,6 +1113,11 @@ def testEnactChangeExistingRow(self):
         self.assertEquals(history_row.data_version, 2)
         self.assertEquals(sc_row.complete, True)
 
+    def testEnactChangeForDeletingExistingRow(self):
```

Comment:
I see you've added a few new tests, but it looks like you're still missing one that verifies that creating a new scheduled change that deletes an existing row will work, unless I'm missing something?